InformationScripting/src/InformationScriptingPlugin.cpp

```diff
@@ -28,15 +28,23 @@
 #include "SelfTest/src/SelfTestSuite.h"
 
 #include "OOInteraction/src/handlers/HStatementItemList.h"
+#include "OOModel/src/declarations/Method.h"
+#include "OOModel/src/declarations/Class.h"
 
 #include "commands/CScript.h"
 #include "helpers/BoostPythonHelpers.h"
+#include "sources/AstSource.h"
+
+#include "visitors/AllNodesOfType.h"
 
 namespace InformationScripting {
 
 bool InformationScriptingPlugin::initialize(Core::EnvisionManager&)
 {
 	BoostPythonHelpers::initializeQStringConverters();
+	AstSource::init();
+	AllNodesOfType<OOModel::Method>::init();
```

Comment:
Nope it's Visitor which visits all nodes of a certain type and stores them in a list. E.g. it is not a filter but rather a data source.